tools/db_bench_tool.cc

```diff
@@ -1062,7 +1062,7 @@ enum RepFactory {
 
 // create Factory for creating S3 Envs
 #ifdef USE_AWS
-rocksdb::Env* CreateAwsEnv(const std::string& dbpath,
+rocksdb::Env* CreateAwsEnv(const std::string& ,
```

Comment:
Typo?

Reply:
The dbpath option is not used, hence why the name is deleted.  This will also lead to errors/warnings if the unused flag is turned on.  How do you want to leave it?